cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConfiguration.java

```diff
@@ -53,6 +53,128 @@ public class KafkaConfiguration extends AbstractConfiguration {
         FORBIDDEN_PREFIX_EXCEPTIONS = AbstractConfiguration.splitPrefixesToList(KafkaClusterSpec.FORBIDDEN_PREFIX_EXCEPTIONS);
     }
 
+    /**
+     * List of configuration options that are relevant to controllers and should be considered when deciding whether
+     * a controller-only node needs to be rolled or not.
+     */
+    private static final Set<String> CONTROLLER_RELEVANT_CONFIGS = Set.of(
```

Comment:
cc @showuon

Reply:
Why do we miss them? The delegation token are not used for anything so why are they relevant. For the others, it would be good if you could provide some more details.